web_modules/App/index.js

```diff
@@ -8,6 +9,21 @@ import Footer from "Footer"
 
 import styles from "./index.css"
 
+const MerriweatherObserver = new FontFaceObserver("Merriweather", {})
+const FiraSansObserver = new FontFaceObserver("Fira Sans", {})
+
+MerriweatherObserver.check().then(() => {
+  document.body.classList.add("merriweather-loaded")
+}, () => {
+  document.body.classList.remove("merriweather-loaded")
+})
+
+FiraSansObserver.check().then(() => {
+  document.body.classList.add("fira-sans-loaded")
+}, () => {
+  document.body.classList.remove("fira-sans-loaded")
+})
+
```

Comment:
Maybe try wrapping this code in an `if (typeof window !== "undefined") {`. If you still have an issue, try replacing the import of fontfaceobserver by a require in this new if window block.